bazel-mode.el

```diff
@@ -0,0 +1,240 @@
+(defun sort-strings-longest-first (l)
```

Comment:
Please start all names with a package prefix. See https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html, especially the second bullet point.

Reply:
Done.